src/WROVER_KIT_LCD.cpp

```diff
@@ -234,10 +234,28 @@ void WROVER_KIT_LCD::scrollTo(uint16_t y) {
     endWrite();
 }
 
+void WROVER_KIT_LCD::setupScrollArea(uint16_t tfa, uint16_t bfa) {
+  startWrite();
+  writeCommand(WROVER_VSCRDEF); // Vertical scroll definition
+  SPI.write(tfa >> 8);
+  SPI.write(tfa);
+  SPI.write((WROVER_HEIGHT - tfa - bfa) >> 8);
+  SPI.write(WROVER_HEIGHT - tfa - bfa);
+  SPI.write(bfa >> 8);
+  SPI.write(bfa);
+  endWrite();
+}
+
 /*
  * Transaction API
  * */
 
+
+/*
+ * Setup a portion of the screen for vertical scrolling
+ * We are using a hardware feature of the display, so we can only scroll in portrait orientation
+ *
+ * */
```

Comment:
I think that you added to the wrong place :) please add it to the header and explain what tfa and bfa are.

Reply:
ok moved this in the .h file